INFO     root:basepage.py:42 等待 账单按钮  元素  ('-android uiautomator', 'new UiSelector().text("账单")') 可见成功。耗时：0:00:02.831000
INFO     root:basepage.py:42 等待 账单按钮  元素  ('-android uiautomator', 'new UiSelector().text("账单")') 可见成功。耗时：0:00:00.278000
INFO     root:basepage.py:79 查找  账单按钮 元素 ('-android uiautomator', 'new UiSelector().text("账单")') 成功
INFO     root:basepage.py:121 点击  账单按钮 元素 ('-android uiautomator', 'new UiSelector().text("账单")') 成功！
INFO     root:basepage.py:42 等待 筛选按钮  元素  ('id', 'com.cashier.jiutongshanghu:id/tv_title_zhangdan') 可见成功。耗时：0:00:01.125000
INFO     root:basepage.py:42 等待 筛选按钮  元素  ('id', 'com.cashier.jiutongshanghu:id/tv_title_zhangdan') 可见成功。耗时：0:00:00.163000
INFO     root:basepage.py:79 查找  筛选按钮 元素 ('id', 'com.cashier.jiutongshanghu:id/tv_title_zhangdan') 成功
INFO     root:basepage.py:121 点击  筛选按钮 元素 ('id', 'com.cashier.jiutongshanghu:id/tv_title_zhangdan') 成功！
INFO     root:basepage.py:42 等待 开始时间  元素  ('id', 'com.cashier.jiutongshanghu:id/nct_start_time') 可见成功。耗时：0:00:00.835000
INFO     root:basepage.py:42 等待 开始时间  元素  ('id', 'com.cashier.jiutongshanghu:id/nct_start_time') 可见成功。耗时：0:00:00.093000
INFO     root:basepage.py:79 查找  开始时间 元素 ('id', 'com.cashier.jiutongshanghu:id/nct_start_time') 成功
INFO     root:basepage.py:121 点击  开始时间 元素 ('id', 'com.cashier.jiutongshanghu:id/nct_start_time') 成功！
INFO     root:orderbook_page.py:792 滑动输入的时间为-->201803111130
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:230 截图成功，截图保存在：C:\Users\Administrator\PycharmProjects\app_automation_sh\output/custom_size_img\时间的年份.png
INFO     root:image_to_text.py:47 文字识别成功,返回值-->2020
INFO     root:orderbook_page.py:805 当前年份是-->2020, 继续滑动...
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:287 页面上下滑动，页面从坐标：1799.16 滑动到坐标：1861.2
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:230 截图成功，截图保存在：C:\Users\Administrator\PycharmProjects\app_automation_sh\output/custom_size_img\时间的年份.png
INFO     root:image_to_text.py:47 文字识别成功,返回值-->2019
INFO     root:orderbook_page.py:805 当前年份是-->2019, 继续滑动...
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:287 页面上下滑动，页面从坐标：1799.16 滑动到坐标：1861.2
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:230 截图成功，截图保存在：C:\Users\Administrator\PycharmProjects\app_automation_sh\output/custom_size_img\时间的年份.png
INFO     root:image_to_text.py:47 文字识别成功,返回值-->2018
INFO     root:orderbook_page.py:805 年份-->2018 滑动选中OK
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:230 截图成功，截图保存在：C:\Users\Administrator\PycharmProjects\app_automation_sh\output/custom_size_img\时间的月份.png
INFO     root:image_to_text.py:47 文字识别成功,返回值-->01
INFO     root:orderbook_page.py:824 当前的月份是-->01, 继续滑动...
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:287 页面上下滑动，页面从坐标：1861.2 滑动到坐标：1799.16
INFO     root:basepage.py:247 当前设备的大小为：{'width': 1080, 'height': 2068}
INFO     root:basepage.py:230 截图成功，截图保存在：C:\Users\Administrator\PycharmProjects\app_automation_sh\output/custom_size_img\时间的月份.png
INFO     root:image_to_text.py:47 文字识别成功,返回值-->02
INFO     root:orderbook_page.py:824 当前的月份是-->02, 继续滑动...
ERROR    root:basepage.py:250 获取设备大小失败。
Traceback (most recent call last):
  File "C:\Users\Administrator\PycharmProjects\app_automation_sh\common\basepage.py", line 246, in get_device_size
    size = self.driver.get_window_size()
  File "C:\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1108, in get_window_size
    size = self.get_window_rect()
  File "C:\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1165, in get_window_rect
    return self.execute(Command.GET_WINDOW_RECT)['value']
  File "C:\Python37\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "C:\Python37\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "C:\Python37\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up